src/Bridge/Symfony/Console/Commands/SearchIndexCleanCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Search\Bridge\Symfony\Console\Commands;
+
+use LoyaltyCorp\Search\Interfaces\Helpers\RegisteredSearchHandlersInterface;
+use LoyaltyCorp\Search\Interfaces\IndexerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class SearchIndexCleanCommand extends Command
+{
+    /**
+     * @var \LoyaltyCorp\Search\Interfaces\IndexerInterface
+     */
+    private $indexer;
+
+    /**
+     * @var \LoyaltyCorp\Search\Interfaces\Helpers\RegisteredSearchHandlersInterface
+     */
+    private $searchHandlers;
+
+    /**
+     * SearchIndexClean constructor.
+     *
+     * @param \LoyaltyCorp\Search\Interfaces\IndexerInterface $indexer
+     * @param \LoyaltyCorp\Search\Interfaces\Helpers\RegisteredSearchHandlersInterface $searchHandlers
+     */
+    public function __construct(IndexerInterface $indexer, RegisteredSearchHandlersInterface $searchHandlers)
+    {
+        $this->indexer = $indexer;
+        $this->searchHandlers = $searchHandlers;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('search:index:clean')
```

Comment:
You should be setting the static $defaultName property, this is an optimisation so that when the kernel is trying to find commands it does not need to instantiate them to find their names, which reduces the amount of dependency resolution required.